src/main/java/racingcar/model/WinnerCar.java

```diff
@@ -0,0 +1,30 @@
+package racingcar.model;
+
+import java.util.Comparator;
+import java.util.List;
+
+public class WinnerCar {
+    private List<String> winnerCarNames;
+    private final int winnerStateValue;
+
+    public WinnerCar(List<Car> cars) {
+        winnerStateValue = cars.stream()
+                .sorted(Comparator.comparing(Car::getCarState).reversed())
+                .toList()
+                .get(0)
+                .getCarState();
```

Comment:
`Car`의 `carState`로 매핑한 다음 최댓값을 구하는 `max()` 함수를 활용한다면 더욱 간단할 것 같아요!🐧
```suggestion
        winnerStateValue = cars.stream()
                .map(Car::getCarState)
                .max(Integer::compareTo)
                .orElseThrow();
```